modules/sdk-coin-tia/test/unit/tia.ts

```diff
@@ -389,7 +389,7 @@ describe('TIA', function () {
       const txnJson = txn.toJson();
       const sendMessage = txnJson.sendMessages[0].value as SendMessage;
       const balance = new BigNumber(testBalance);
-      const gasAmount = new BigNumber(7000);
+      const gasAmount = new BigNumber(30000);
```

Comment:
I think it'd be better if we actually referred to these numbers from the coin files instead of hardcoding it just because were changing it a lot